app/assets/stylesheets/forever_style_guide/modules/_pnav.scss

```diff
@@ -0,0 +1,464 @@
+
+$f-pnav__main--zindex: 1030;
+$f-pnav__side--width: 320px;
+$f-pnav__side--zindex: 1032;
+
+@mixin deals_link_base() {
+  color: $color-forever_orange;
+
+  &:hover {
+    color: $color-forever_blue;
+  }
+
+  &::after {
+    position: relative;
+    top: -5px;
+    right: -2px;
+    font-family: FontAwesome;
+    font-size: 10px;
+    content: '\f005';
+  }
+}
+
+.f-pnav {
+  width: 100%;
+  padding: 0 10px;
+  box-shadow: 0 3px 5px rgba(0, 0, 0, 0.1);
+  background-color: $color-white;
```

Comment:
Could probably just use a color_block class in the markup.

Reply:
One could go either way on this for the sole reason that markup is shared across 2 apps. Seemed DRYer given that to set background-color as part of the class as opposed to having it in the markup on the consumer side of things.

Also, in this case, the styleguide is being explicit in saying "I want this thing to have a background-color of ...` versus relying on implicit knowledge of the consumer to say `Styleguide wants me to make this have a background-color of ...`

Reply:
yea that makes sense.